adapters/appnexus/appnexus.go

```diff
@@ -455,9 +450,12 @@ func appendMemberId(uri string, memberId string) string {
 // Builder builds a new instance of the AppNexus adapter for the given bidder with the given config.
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
 	bidder := &adapter{
-		URI:            config.Endpoint,
-		iabCategoryMap: loadCategoryMapFromFileSystem(),
-		hbSource:       resolvePlatformID(config.PlatformID),
+		URI: config.Endpoint,
+		iabCategoryMap: map[string]string{
+			"1": "IAB20-3",
+			"9": "IAB5-3",
+		},
+		hbSource: resolvePlatformID(config.PlatformID),
```

Comment:
the code formatting looks off to me maybe because of map literal. Should we create a constant instead since this map value will not change during the execution of the program?

Reply:
> the code formatting looks off to me maybe because of map literal

This might be related to way diff is viewed in browser.

> Should we create a constant instead since this map value will not change during the execution of the program?
Map cannot be declared as constant.

Refer https://go.dev/doc/effective_go#constants

`Constants in Go are just that—constant. They are created at compile time, even when defined as locals in functions, and can only be numbers, characters (runes), strings or booleans`